bittensor/metagraph.py

```diff
@@ -858,43 +838,28 @@ def load_from_path(self, dir_path: str) -> "metagraph":
             contain valid data for the metagraph. It is essential to ensure that the directory path and the
             state files within it are accurate and consistent with the expected metagraph structure.
         """
-        graph_file = latest_block_path(dir_path)
-        state_dict = torch.load(graph_file)
-        self.n = torch.nn.Parameter(state_dict["n"], requires_grad=False)
-        self.block = torch.nn.Parameter(state_dict["block"], requires_grad=False)
-        self.uids = torch.nn.Parameter(state_dict["uids"], requires_grad=False)
-        self.stake = torch.nn.Parameter(state_dict["stake"], requires_grad=False)
-        self.total_stake = torch.nn.Parameter(
-            state_dict["total_stake"], requires_grad=False
-        )
-        self.ranks = torch.nn.Parameter(state_dict["ranks"], requires_grad=False)
-        self.trust = torch.nn.Parameter(state_dict["trust"], requires_grad=False)
-        self.consensus = torch.nn.Parameter(
-            state_dict["consensus"], requires_grad=False
-        )
-        self.validator_trust = torch.nn.Parameter(
-            state_dict["validator_trust"], requires_grad=False
-        )
-        self.incentive = torch.nn.Parameter(
-            state_dict["incentive"], requires_grad=False
-        )
-        self.emission = torch.nn.Parameter(state_dict["emission"], requires_grad=False)
-        self.dividends = torch.nn.Parameter(
-            state_dict["dividends"], requires_grad=False
-        )
-        self.active = torch.nn.Parameter(state_dict["active"], requires_grad=False)
-        self.last_update = torch.nn.Parameter(
-            state_dict["last_update"], requires_grad=False
-        )
-        self.validator_permit = torch.nn.Parameter(
-            state_dict["validator_permit"], requires_grad=False
-        )
-        self.uids = torch.nn.Parameter(state_dict["uids"], requires_grad=False)
+        graph_filename = latest_block_path(dir_path)
+        with open(graph_filename, "rb") as graph_file:
+            state_dict = pickle.load(graph_file)
```

Comment:
please prepare changes list with migration steps required

it seems we are missing a routine here where it detects that this dump was made using previous, torch version, and it converts it to new, numpy version

or make it a manual step (still code should be provided IMO)

Reply:
added fallback to also work on previous torch saves